frontend/src/container/AdvancedSearch.jsx

```diff
@@ -52,7 +62,55 @@ export class AdvancedSearch extends React.Component {
     this.buildBreadcrumbs()
   }
 
+  setHeadTitle = () => {
+    const { props } = this
+    const headTitle = buildHeadTitle(
+      [`${props.t('Search results')} : ${parseSearchUrl(qs.parse(props.location.search)).searchedKeywords}`]
+    )
+
+    props.dispatch(setHeadTitle(headTitle))
+  }
+
+  buildBreadcrumbs = () => {
+    const { props } = this
+
+    props.dispatch(setBreadcrumbs([{
+      link: PAGE.SEARCH_RESULT,
+      type: BREADCRUMBS_TYPE.CORE,
+      label: props.t('Search results'),
+      isALink: true
+    }]))
+  }
+
   componentDidMount () {
+    const { props } = this
+    const searchObject = parseSearchUrl(qs.parse(props.location.search))
+
+    if (searchObject.searchType !== ADVANCED_SEARCH_TYPE.CONTENT) {
```

Comment:
I'm confused by the `!==`
in the if you actually do the `searchType: ADVANCED_SEARCH_TYPE.CONTENT`

Reply:
searchObject is the object coming from the url (maybe I rename it to urlSearchObject), so the idea is: if it's not the search type from the url, it does the redux set anyway (to run the 3 searches in parallel) with all the same settings except the type and the current page